Source/ACE/Managers/GuidManager.cs

```diff
@@ -45,6 +53,16 @@ public static uint NewPlayerGuid()
         }
 
         /// <summary>
+        /// Returns New Admin Guid
+        /// </summary>
+        /// <returns></returns>
+        public static uint NewAdminGuid()
```

Comment:
No doubt here and I also wonder if GuidManager is generating the ids for what eventually is stored in ace_shard when saving inventory or whatever, if so.. how does it know which ids are taken up (it doesn't)

I just made sure everything that generates ids is using the same function now. I would most certainly need yours or others assistance to make this thing function as it appears to be meant for